data/build.gradle

```diff
@@ -1,27 +1,18 @@
 plugins {
-    id 'com.android.application'
+    id 'com.android.library'
```

Comment:
only UI-like modules should be declared as `application`, the rest can be `library`